packages/salesforcedx-vscode-soql/src/queryResultsView/media/queryDataViewController.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+(function() {
+  /* interface queryData {
+    done: boolean;
+    nextRecordsUrl?: string;
+    totalSize: number;
+    records: T[];
+  } */
+  const vscode = acquireVsCodeApi();
+
+  function updateUIWith(queryData, documentName) {
+    // Display the .soql file name as the title
+    const titleEl = document.getElementById('webview-title');
+    titleEl.innerText = documentName;
+
+    const totalRecordsSizeEl = document.getElementById('total-records-size');
+    totalRecordsSizeEl.innerText = `Returned ${queryData.records.length} of ${queryData.totalSize} total records`;
+
```

Comment:
@ekapner I added some new text to the table webview, this is modeled after the text currently in Workbench. Do you want to make any changes?

The total number represents the total number of records that match the query in the org, where as the "returned" value is the number of the total records returned by the API, capped at 2000.

So it's possible they would get "Returned 2000 of 3750 total records" etc.
![image](https://user-images.githubusercontent.com/25188632/94171624-b40ee280-fe4e-11ea-962b-208d4047b7e1.png)